vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/text_processing/embedding_processing.cc

```diff
@@ -0,0 +1,131 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/text_processing/embedding_processing.h"
+
+#include <algorithm>
+#include <utility>
+#include <vector>
+
+#include "base/base64.h"
+#include "base/check.h"
+#include "base/strings/string_split.h"
+#include "base/values.h"
+#include "bat/ads/internal/base/crypto/crypto_util.h"
+#include "bat/ads/internal/base/logging_util.h"
+#include "bat/ads/internal/base/strings/string_html_parse_util.h"
+#include "bat/ads/internal/base/strings/string_strip_util.h"
+#include "bat/ads/internal/ml/data/text_data.h"
+#include "bat/ads/internal/ml/data/vector_data.h"
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_info.h"
+#include "bat/ads/internal/ml/pipeline/pipeline_util.h"
+#include "bat/ads/internal/ml/pipeline/text_processing/embedding_data.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
+
+namespace ads {
+namespace ml {
+namespace pipeline {
+
+// static
+std::unique_ptr<EmbeddingProcessing> EmbeddingProcessing::CreateFromValue(
+    base::Value resource_value,
+    std::string* error_message) {
+  DCHECK(error_message);
+
+  auto text_processing = std::make_unique<EmbeddingProcessing>();
+  if (!text_processing->FromValue(std::move(resource_value))) {
+    *error_message = "Failed to parse text embedding pipeline JSON";
+    return {};
+  }
+
+  return text_processing;
+}
+
+bool EmbeddingProcessing::IsInitialized() const {
+  return is_initialized_;
+}
+
+EmbeddingProcessing::EmbeddingProcessing() : is_initialized_(false) {}
```

Comment:
`= default;` please set is_initialized_ = false in .h

Reply:
Updated .cc file. 
.h file should already be correct